pg_search/src/postgres/storage/buffer.rs

```diff
@@ -170,6 +170,10 @@ impl Page<'_> {
         unsafe { self.pg_page.read_item(offno) }
     }
 
+    pub fn read_pg_item(&self, offno: pg_sys::OffsetNumber) -> Option<PgItem> {
+        unsafe { self.pg_page.read_pg_item(offno) }
+    }
+
```

Comment:
This isn't necessary.  There's `Page::read_item<T>()` that already exists and should be used